docs/shared/packages/angular/angular-nx-version-matrix.md

```diff
@@ -54,3 +54,4 @@ Below is a reference table that matches versions of [Angular Rspack](/recipes/an
 | Angular Rspack | Angular     | Nx                  |
 | -------------- | ----------- | ------------------- |
 | ~20.6.0        | **~19.2.0** | >= 20.6.0 <= latest |
+| ~20.7.0        | **~19.2.0** | >= 20.8.1 <= latest |
```

Comment:
It'd be better to invert the order here:
```suggestion
| ~20.7.0        | **~19.2.0** | >= 20.8.1 <= latest |
| ~20.6.0        | **~19.2.0** | >= 20.6.0 <= latest |
```

Reply:
ah good idea 👍